src/NuGet.Core/NuGet.Commands/RestoreCommand/MSBuildProjectReferenceProvider.cs

```diff
@@ -191,7 +191,7 @@ private PackageSpec GetPackageSpec(string msbuildProjectPath)
             {
                 if (File.Exists(path))
                 {
-                    result = JsonPackageSpecReader.GetPackageSpec(projectName, path);
+                    result = JsonPackageSpecReader.GetPackageSpec(projectName, path, string.Empty);
```

Comment:
Also I'd prefer having null as a default value. It looks somewhat clearer to me.